extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -71,6 +71,12 @@ export class BaseCarousel extends AMP.BaseElement {
     // TODO(erwinm): Does label need i18n support in the future? or provide
     // a way to be overridden.
     this.prevButton_.setAttribute('aria-label', 'Previous item in carousel');
+    this.prevButton_.setAttribute('tabindex', 0);
+    this.prevButton_.onkeydown = event => {
+      if (event.keyCode == 13 /* enter */ || event.keyCode == 32 /* space */) {
```

Comment:
I suggest creating an issue to add `keycodes.js` with the enum and use it everywhere ( there are many other places that check keycode, specially for ESC, if we do the enum thing, better to do it everywhere)